### ISSUE TITLE
Where to install Muse SDK for Windows

### ISSUE THREAD

**Issue by BioMycoBit** [rbuttery/muse_eeg_live_brainwave_app#1]
Purchased a Muse Headband recently and interested in trying out the app.  I have downloaded the SDK for windows but I do not see where the files need to be copied to use your app.

Also, are you able to use your built-in Bluetooth hardware, or are you needing a dedicated BLE112 dongle?

Thanks!
Brandon

**Comment by rbuttery** [comment]
Are you sure you have the Muse SDK? It should work with the headband's bluetooth connection, and of course a bluetooth connection on the Windows device running the Muse SDK.  

Once you got that running you can setup the python environment and run the muse_receiver_app.py, ensuring that the port is the same as the SDK (if I recall correctly I think its port 7000).

Happy to help if you get stuck!

**Comment by BioMycoBit** [comment]
I appreciate the help. 

I was able to retrieve the Muse SDK via the OneDrive link provided by Interaxon in the SDK email.

SDK Retrieved
- libmuse_windows_7.2.2

Have no idea where / how to install it to use with your app.

**Comment by rbuttery** [comment]
Ah Libmuse is a different SDK - Muse has lots of research and SDK kits. Did you apply via https://choosemuse.com/pages/developers? 

To be fully transparent I work for Muse so I was able to use this SDK called Muse Direct - but now I'm looking into it a bit, I'm seeing the license is restrictive for this specific SDK. 

You are able to apply for it, however I'm unaware of how that process works and there is likely a cost associated with it (same page refereced above - https://choosemuse.com/pages/developers).

One option that is likely easier for hobby projects is using https://mind-monitor.com/ - its a well-known 3rd party dev who made an app to use the Muse SDK. It will work perfectly with my repo, you just need to ensure you're on the same port in both  the app and the SDK (or in this case, the Mind Monitor app).

Mind Monitor is a 1-time purchase so there's no monthly fee which is nice.

**Comment by BioMycoBit** [comment]
Yeah, I applied via the developers link.   It seems like it might have been automatic since my approval was within an hour or so after hours.

Does the Muse Direct SDK have access to Heart Rate as well?

**Comment by rbuttery** [comment]
Well the SDK allows for access to the individual signals from the band, however they are unprocessed signals so the heart-rate data is not in BPM, but rather the raw signal data itself. Muse Direct is a direct access to the raw signals, which need to be processed to extract any meaningful insights. 

So these signals would be your 6 EEG channels, PPG, Gyro & Accelerometer. Also has stuff like battery level, the "horse shoe" which just says which sensors are properly connected.

**Comment by rbuttery** [comment]
The signals are processed into the scores in the Muse App (calm, stillness, body, heart) and any new ones like Alpah Peak. Muse Direct is basically just the headband data, no signal processing at all.
